notification/cmd/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"os/signal"
+	"syscall"
+
+	"github.com/GrabItYourself/giys-backend/lib/logger"
+	"github.com/GrabItYourself/giys-backend/lib/rabbitmq"
+	"github.com/GrabItYourself/giys-backend/lib/rabbitmq/types"
+	"github.com/GrabItYourself/giys-backend/notification/internal/config"
+	"github.com/GrabItYourself/giys-backend/notification/internal/handler"
+	"github.com/pkg/errors"
+	amqp "github.com/rabbitmq/amqp091-go"
+)
+
+func main() {
+	// Context
+	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGTERM, syscall.SIGINT)
+	defer cancel()
+
+	// Config
+	conf := config.InitConfig()
+
+	// Logger
+	logger.InitLogger(&conf.Log)
+
+	// Connect to RabbitMQ
+	conn, err := amqp.Dial(conf.RabbitMQ.URL)
+	if err != nil {
+		logger.Fatal(errors.Wrap(err, "Failed to connect to RabbitMQ").Error())
```

Comment:
we decided to switch to `logger.Panic` instead to trigger `defer` clauses (Fatal just exits immediately)